Makefile.toml

```diff
@@ -25,6 +25,16 @@ private = true
 condition = { env = { "TEST_SUITE" = "platinum" }, env_not_set = ["ELASTICSEARCH_URL"] }
 env = { "ELASTICSEARCH_URL" = "https://elastic:changeme@localhost:9200" }
 
+[tasks.download-specs]
+category = "Elasticsearch"
+description = '''Download Rest API specs and YAML tests'''
+private = true
+command = "cargo"
+# cargo-make insists on installing cargo-xtask and ignores .cargo/config.toml
```

Comment:
@swallez you can disable it by adding:
install_crate = false
see more details at:
https://github.com/sagiegurari/cargo-make#cargo-plugins

Reply:
Thanks for the hint @sagiegurari!